pandas/io/formats/terminal.py

```diff
@@ -48,6 +48,23 @@ def get_terminal_size():
     return tuple_xy
 
 
+def is_terminal():
+    """
+    Detect if Python is running in a terminal.
+
+    Returns True if Python is running in a terminal or False if not.
+    """
+    try:
+        ip = get_ipython()
+    except NameError:  # assume standard Python interpreter in a terminal
+        terminal = True
+        if hasattr(ip, 'kernel'):  # IPython as a Jupyter kernel
```

Comment:
Does this need to be an `else` block of the `try / except`?

Reply:
Of course...